src/study_interface.jl

```diff
@@ -899,6 +899,24 @@ Dict{String, Vector} with 13 entries:
 """
 function get_series end
 
+
+"""
+function get_series(
+    data::Data,
+    collection::String,
+    attribute::String;
+    kws...
+)
+
+Retrieves a GrafTable object with the values for 'attribute' in all elements in 'collection' from a Graf file.
+
+Example
+```
+julia> PSRI.get_graf_series(data, "PSRDemandSegment", "HourDemand")
```

Comment:
Lets not use HourDemand as the example.
We can use "Demand" (the non hourly one)

Reply:
no correction?

Reply:
I am changing it to the same example we are now using in the documentation with PSRGasEmission where `EmissionCost` is indexed by `DateEmissionCost`. Is that ok?